crates/node_binding/src/plugins/interceptor.rs

```diff
@@ -1419,7 +1419,7 @@ impl NormalModuleFactoryResolve for NormalModuleFactoryResolveTap {
   }
 
   fn stage(&self) -> i32 {
```

Comment:
should change to f64 too

Reply:
yes, but there ara hard implications when I change it :) Let me show you @hardfist

Reply:
@hardfist When I change it, I see this while bouilding:
![obraz](https://github.com/user-attachments/assets/f7b0789b-66f9-45ca-8b0e-76c5669b48e5)
And it seems I need to align way more code...